ffbs-parker-nextnode/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ffbs-parker-nextnode
+PKG_VERSION:=1
+
+PKG_MAINTAINER:=Chris Fiege <chris@tinyhost.de>
+PKG_LICENSE:=MIT
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/ffbs-parker-nextnode
+  TITLE:=gluon-nextnode config for parker
+  DEPENDS:=+ffbs-parker-nodeconfig
```

Comment:
As https://github.com/freifunk-gluon/community-packages/pull/142 is still WIP, is this name fixed?

Reply:
I would rather remove the depends here, and add a dependency to ffbs-parker-nextnode on `ffbs-gluon-mesh-vpn`, as this package is quite standalone, but the other can only be used efficiently in combination with this package.

Reply:
I don't see why anyone would use this package without `ffbs-parker-nextnode` (aka the *ffbs-parker-vpn-provider*). But you are right regarding the direction of the dependency.

Reply:
Let's change it  then and rename the DEPENDS to the new name `ffbs-mesh-vpn-parker `

Reply:
Droped the `DEPENDS` here and will add it to `ffbs-mesh-vpn-parker`.